qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/commons/datamodel/resolvers/ValueResolverKind.java

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
+* Copyright (c) 2022 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.qute.commons.datamodel.resolvers;
+
+/**
+ * Represents the kind of a value resolver.
+ * 
+ * @author datho7561
+ */
+public enum ValueResolverKind {
+	TemplateData, TemplateEnum, TemplateExtensionOnMethod, TemplateExtensionOnClass, TemplateGlobal, InjectedBean;
+
+	public String toString() {
```

Comment:
Please manage this enum like https://github.com/redhat-developer/quarkus-ls/blob/afbcdfea4d24ac69b958613e755f34966d831e9e/qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/commons/JavaTypeKind.java#L36 with forValue. It gives the capability to serialize enum as int.